IBPSA/BoundaryConditions/WeatherData/BaseClasses/getTimeSpanWeatherData.mo

```diff
@@ -0,0 +1,87 @@
+within IBPSA.BoundaryConditions.WeatherData.BaseClasses;
+function getTimeSpanWeatherData
```

Comment:
`WeatherData` could be deleted from the name to keep it brief, as the function is in the `WeatherData` package.

Reply:
Ok. Renamed to getTimeSpanTMY3.